Doc/library/asyncio-task.rst

```diff
@@ -282,6 +282,24 @@ Creating Tasks
       Added the *context* parameter.
 
 
+Task Cancellation
+=================
+
+Tasks can easily and safely be cancelled.
+When a task is cancelled, asyncio will raise a :exc:`asyncio.CancelledError`
+into the task at the next opportunity.
```

Comment:
Either "throw ... into the task" or "raise ... in the task". Also, no "a".

Reply:
I like `raise in the task` better. How about just `CancelledError will be raised` instead of `asyncio will raise`?

Reply:
Sure!